docssrc/src/permissions.md

```diff
@@ -117,3 +117,82 @@ Now we declare our command with arguments. We use a `PlayerArgument` and apply t
 > As you can see, there are multiple ways of applying permissions to commands with arguments. In the `/god` command shown above, the permission was applied to the whole command. In the `/kill` command shown above, the permission was applied to the argument.
 >
 > There's not really much difference between the two methods, but I personally would use _argument permissions_ as it has greater control over arguments.
+
+-----
+
+## Child Base Permissions
+### Example - /economy command with argument permissions
+Child Based Permissions allow you to group permissions together, we achieve this by laying out our permission groups in the `plugin.yml` file which Bukkit registers as valid permissions. When the CommandAPI checks if our player has a permission Bukkit considers if they have the child of a permission as well. This not only keeps permissions easier to manage, it also makes your code cleaner and gives you a nice place to layout out all of your permissions, what they do and what other permissions inherit them.
+
+For example, say we're registering a command `/economy`:
+
+```mccmd
+/economy                         - shows your own balance                 | economy.self
+/economy <target>                - shows you another players balance      | economy.other
+/economy give  <target> <amount> - gives the target a set amount of money | economy.admin.give
+/economy reset <target> <amount> - resets the targets balance             | economy.admin.reset
+```
+
+We first declare the command as normal. Nothing fancy is going on here:
+
+<div class="multi-pre">
+
+```java,Java
+{{#include ../../commandapi-documentation-code/src/main/java/dev/jorel/commandapi/examples/java/Examples.java:permissions4_1)}}
```

Comment:
```
permissions4_1)}}
```
There's a `)` character here which breaks this link, preventing a code block from showing